Makefile.dev

```diff
@@ -242,6 +242,14 @@ build-all: build-app create-build-config build-container set-app-image build-ima
 deploy-changes:
 	./deploy-changes.sh
 
+setup-deps:
+	@echo "Enabling required repositories..."
+	sudo dnf install -y epel-release dnf-plugins-core
```

Comment:
is that needed?

Reply:
yes, without this, dnf can’t find the vsomeip3 packages and the install fails. I think it properly sets up the COPR repo with the correct keys and enables it, unlike just curling the .repo file.

Reply:
it worked for me without that yesterday. not htat it matters much....